sherpa/astro/data.py

```diff
@@ -72,7 +72,7 @@
 coordinate 3150,4515:
 
 >>> from sherpa.astro.io import read_image
->>> img = read_image('clus.fits')
+>>> img = read_image('sherpa-test-data/sherpatest/image2.fits')
```

Comment:
works when sherpa-test-data is installed and will hopefully indicate to a user "put your own file name here".

Reply:
Which is something we can't resolve in this PR but I wanted to mention it.